llvm/test/Transforms/JumpThreading/select.ll

```diff
@@ -21,7 +21,7 @@ declare void @quux()
 ; booleans where at least one operand is true/false/undef.
 
 ;.
-; CHECK: @[[ANCHOR:[a-zA-Z0-9_$"\\.-]+]] = constant [3 x ptr] [ptr blockaddress(@test_indirectbr, [[L1:%.*]]), ptr inttoptr (i32 1 to ptr), ptr blockaddress(@test_indirectbr, [[L3:%.*]])]
+; CHECK-BPI: @anchor = constant [3 x ptr] [ptr blockaddress(@test_indirectbr, %L1), ptr inttoptr (i32 1 to ptr), ptr blockaddress(@test_indirectbr, %L3)]
```

Comment:
Why is this CHECK-BPI now?

Reply:
I have no idea about it, `update_test_checks` replaces it even without this patch. Seems the default behavior of multiple prefixes changes for globals.

Reply:
It looks like this is somehow related to the debug output. I think you can modify the RUN lines to look like this:
```
; RUN: opt -S -passes="jump-threading" < %s 2>&1 | FileCheck %s
; RUN: opt -S -passes="require<branch-prob>,jump-threading" -debug-only=branch-prob -disable-output < %s 2>&1 | FileCheck -check-prefix=CHECK-BPI %s
```

Reply:
Thanks, it works now.